TranslatorReasonerAPI.yaml

```diff
@@ -290,7 +290,9 @@ components:
         - edges
     QNode:
       type: object
-      description: A node in the QueryGraph
+      description: A node in the QueryGraph used represent an entity in a
+        query. If a CURIE is not specified, any nodes matching the type
+        of the QNode will be returned is the Results.
```

Comment:
Another little typo: Did you mean to say "...will be returned [in] the Results"?